drag-arrange.js

```diff
@@ -84,9 +81,11 @@
       var $this = $(this);
 
       if (dragSelector) {
-        $this.on(dragEvents.START + eventNamespace, dragSelector, dragStartHandler);
+        $this.on(dragEventsMouse.START + eventNamespace, dragSelector, dragStartHandler);
```

Comment:
@vishalmindtickle You mean, for touch-only devices (smartpones/tablets)? Hmm, could be...
Any idea how I could check this?